metadata-ingestion/src/datahub/ingestion/source/elastic_search.py

```diff
@@ -457,6 +470,22 @@ def _extract_mcps(
                     ),
                 ),
             )
+        cat_response = self.client.cat.indices(
+            index=index, params={"format": "json", "bytes": "b"}
+        )
+        if len(cat_response) == 1:
```

Comment:
What if len is not 1?
Shouldn't we write some warning or something?

Reply:
I think you should use get Index Api instead of `cat`

This is it's doc on Elastic
```
cat APIs are only intended for human consumption using the command line or Kibana console. They are not intended for use by applications. For application consumption, use the [get index API](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-get-index.html).
```

Reply:
Added warning for when there is no response